### DISCUSSION TITLE
Can not deloy by docker to heroku

### DISCUSSION THREAD

**Discussion by maearon** [saleor/saleor#11249] [Deployment]
I config and push to heroku but it crash

sleepy-dawn-64450.herokuapp.com

**Comment by karolkielecki** [comment]
Hi @maearon, Did you follow https://docs.saleor.io/docs/3.x/developer/running-saleor/heroku? 
Have you considered using our Cloud solution instead of Heroku? You can create a free developer account using CLI: https://github.com/saleor/saleor-cli
